dotcom-rendering/src/server/prod-server.ts

```diff
@@ -52,79 +52,43 @@ export const prodServer = (): void => {
 		app.use('/assets', express.static(__dirname));
 	}
 
-	app.post('/Article', logRenderTime, renderArticle);
-	app.post('/AMPArticle', logRenderTime, renderAMPArticle);
-	app.post('/Interactive', logRenderTime, renderInteractive);
-	app.post('/AMPInteractive', logRenderTime, renderAMPArticle);
-	app.post('/Blocks', logRenderTime, renderBlocks);
-	app.post('/KeyEvents', logRenderTime, renderKeyEvents);
-	app.post('/Front', logRenderTime, renderFront);
-	app.post('/FrontJSON', logRenderTime, renderFrontJson);
+	app.post('/Article', logRenderTime, handleArticle);
+	app.post('/AMPArticle', logRenderTime, handleAMPArticle);
+	app.post('/Interactive', logRenderTime, handleInteractive);
+	app.post('/AMPInteractive', logRenderTime, handleAMPArticle);
+	app.post('/Blocks', logRenderTime, handleBlocks);
+	app.post('/KeyEvents', logRenderTime, handleKeyEvents);
+	app.post('/Front', logRenderTime, handleFront);
+	app.post('/FrontJSON', logRenderTime, handleFrontJson);
 
 	// These GET's are for checking any given URL directly from PROD
 	app.get(
 		'/Article',
 		logRenderTime,
 		getContentFromURLMiddleware,
-		async (req: Request, res: Response) => {
-			// Eg. http://localhost:9000/Article?url=https://www.theguardian.com/commentisfree/...
-			try {
-				return renderArticle(req, res);
-			} catch (error) {
-				console.error(error);
-			}
-		},
+		handleArticle,
 	);
+	app.use('/ArticleJson', handleArticleJson);
 
 	app.get(
 		'/AMPArticle',
 		logRenderTime,
 		getContentFromURLMiddleware,
-		async (req: Request, res: Response) => {
-			// Eg. http://localhost:9000/AMPArticle?url=https://www.theguardian.com/commentisfree/...
-			try {
-				return renderAMPArticle(req, res);
-			} catch (error) {
-				console.error(error);
-			}
-		},
-	);
-
-	app.get(
-		'/Front',
-		logRenderTime,
-		// TODO: ensure getContentFromURLMiddleware supports fronts
-		getContentFromURLMiddleware,
-		async (req: Request, res: Response) => {
```

Comment:
This used to be an `async` function. Do we need to keep this behaviour? And if yes, are we keeping this behaviour somewhere?

Reply:
I’m not entirely sure… I’m assuming this has to do with the `getContentFromURLMiddleware`, maybe? We could keep it as-is for now and extract that to a subsequent PR.